client/extensions/wp-job-manager/components/settings/job-submission/index.jsx

```diff
@@ -1,5 +1,197 @@
+						<FormSettingExplanation isIndented>
+							{ translate( 'Limits job listing submissions to registered, logged-in users.' ) }
+						</FormSettingExplanation>
+
+						<ReduxFormToggle
+							name="job_manager_enable_registration"
+							text={ translate( 'Enable account creation during submission' ) } />
+						<FormSettingExplanation isIndented>
+							{ translate( 'Includes account creation on the listing submission form, to allow ' +
+								'non-registered users to create an account and submit a job listing simultaneously.' ) }
+						</FormSettingExplanation>
+
+						<ReduxFormToggle
+							name="job_manager_generate_username_from_email"
+							text="Generate usernames from email addresses" />
+						<FormSettingExplanation isIndented>
+							{ translate( 'Automatically generates usernames for new accounts from the registrant\'s ' +
+								'email address. If this is not enabled, a "username" field will display instead.' ) }
+						</FormSettingExplanation>
+					</FormFieldset>
+
+					<FormFieldset>
+						<FormLabel>
+							{ translate( 'Role' ) }
+						</FormLabel>
+						<ReduxFormSelect name="job_manager_registration_role">
+							<option value="editor">{ translate( 'Editor' ) }</option>
+							<option value="author">{ translate( 'Author' ) }</option>
+							<option value="contributor">{ translate( 'Contributor' ) }</option>
+							<option value="subscriber">{ translate( 'Subscriber' ) }</option>
+							<option value="teacher">{ translate( 'Teacher' ) }</option>
+							<option value="employer">{ translate( 'Employer' ) }</option>
+							<option value="customer">{ translate( 'Customer' ) }</option>
+							<option value="shop_manager">{ translate( 'Shop manager' ) }</option>
+						</ReduxFormSelect>
+						<FormSettingExplanation>
+							{ translate( 'Any new accounts created during submission will have this role.' ) }
+						</FormSettingExplanation>
+					</FormFieldset>
+				</Card>
+			</form>
+
+			<form>
+				<SectionHeader label={ translate( 'Approval' ) }>
+					<FormButton compact primary />
+				</SectionHeader>
+				<Card>
+					<FormFieldset>
+						<ReduxFormToggle
+							name="job_manager_submission_requires_approval"
+							text={ translate( 'Require admin approval of all new listing submissions' ) } />
+						<FormSettingExplanation isIndented>
+							{ translate( 'Sets all new submissions to "pending." They will not appear on your ' +
+								'site until an admin approves them.' ) }
+						</FormSettingExplanation>
+
+						<ReduxFormToggle
+							name="job_manager_user_can_edit_pending_submissions"
+							text="Allow editing of pending listings" />
+						<FormSettingExplanation isIndented>
+							{ translate( 'Users can continue to edit pending listings until they are approved by an admin.' ) }
+						</FormSettingExplanation>
+					</FormFieldset>
+				</Card>
+			</form>
+
+			<form>
+				<SectionHeader label={ translate( 'Listing Duration' ) }>
```

Comment:
Hi! I've found a possible matching string that has already been translated **32** times:
`translate( 'Duration' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).